services-js/public-notices/src/components/NoticeDetail.stories.tsx

```diff
@@ -8,7 +8,7 @@ import NoticeDetail from './NoticeDetail';
 
 const NOTICES: Notice[] = require('../../fixtures/notices.json');
 const LONG_NOTICE = NOTICES[1];
-const CANCELLED_NOTICE = NOTICES[17];
+const CANCELED_NOTICE = NOTICES[17];
```

Comment:
Why do we gotta hate on the queen’s english?? 😜

Reply:
![colonialsoldierwithprisoner](https://user-images.githubusercontent.com/260837/51042205-ba313280-1589-11e9-97e8-9ea71406947d.gif)